certified-connectors/sendSMS/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "Value-From-Your-App-Registration",
+          "clientSecret": "Value-From-Your-App-Registration",
+          "scopes": ["https://api.sendsms.ro/.default"],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False"
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://api.sendsms.ro/oauth2/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://api.sendsms.ro/oauth2/token"
+            },
+            "refreshUrl": {
+              "value": "https://api.sendsms.ro/oauth2/refresh_token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#ffffff",
```

Comment:
please make sure you are using the iconBrandColor for your connector. Icon color must contain a non-transparent, non-white color (#ffffff) background and not-default color (#007ee5) that matches your specified icon background color. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector).